OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -103,7 +103,7 @@ public override Activity Tick(Actor self)
 				foreach (var n in notify)
 					n.MovingToResources(self, closestHarvestablePosition.Value, this);
 
-				return ActivityUtils.SequenceActivities(mobile.MoveTo(closestHarvestablePosition.Value, 1), new HarvestResource(self), this);
+				return ActivityUtils.SequenceActivities(mobile.MoveTo(closestHarvestablePosition.Value, 2), new HarvestResource(self), this);
```

Comment:
Besides, there are many reasons that prevent harvester reaching the ore in the field, like enemy units or ally units blocking the way. I'd say it is safer to have this parameter 2, which covers the diagonal adjacency, unlike 1.

If you set this to 0 you'll see the harvesters getting deadlocked in horizontal/vertical positions.
Currently, =1 doesn't cover diagonal cells and that's why the harvesters are always in diagonal positions when they get stuck in the field. (MoveTo parameter of =1 translates to 1024 so it can't cover WDist of sqrt(2))